### DISCUSSION TITLE
EqualStreetNames.Wrocław

### DISCUSSION THREAD

**Discussion by ArturLange** [EqualStreetNames/equalstreetnames#764] [New city]
Hello, I've created a repository for [Wrocław, Poland](https://github.com/EqualStreetNames/equalstreetnames-wroclaw).
MAPBOX_TOKEN should be set up, it just needs the submodule and stuff.

Wikidata linking is not very complete yes, but I'm going to fill the gaps in the next few days.
Thanks in advance.

**Comment by jbelien** [comment]
Great work @ArturLange !

🚀 It's now deployed: https://wroclaw.equalstreetnames.eu/
